keyboards/ergodox_ez/keymaps/hacker_dvorak/keymap.c

```diff
@@ -0,0 +1,797 @@
+#include QMK_KEYBOARD_H
+#include "ergodox_ez.h"
+#include "action_layer.h"
+#include "keymap_dvorak.h"
+#include "sendstring_dvorak.h"
+#include "keymap_plover.h"
+#include "keymap_plover_dvorak.h"
+
+
+#define SCTL(kc) LSFT(LCTL(kc))
+#define MS_CC    MAGIC_SWAP_CONTROL_CAPSLOCK
+#define MU_CC    MAGIC_UNSWAP_CONTROL_CAPSLOCK
+#define MS_GE    MAGIC_SWAP_GRAVE_ESC
+#define MU_GE    MAGIC_UNSWAP_GRAVE_ESC
+#define MT_NKRO  MAGIC_TOGGLE_NKRO
+#define MT_AG    MAGIC_TOGGLE_ALT_GUI
+#define MY_GUI   MAGIC_UNNO_GUI
+#define MN_GUI   MAGIC_NO_GUI
+
+// Layers
+enum layers {
+    DVORAK_US = 0,
+    PLOVER_US,
+    GAMING_US,
+    ARROWS_US,
+    MOUSE_US,
+    NUMPAD_US,
+    LAYERS_US,
+    DVORAK_DV,
+    PLOVER_DV,
+    GAMING_DV,
+    ARROWS_DV,
+    MOUSE_DV,
+    NUMPAD_DV,
+    LAYERS_DV,
+    MEDIA_FN,
+    HYPER_FN,
+    MEH_FN,
+    MEH_FN_PLUS,
+    MAGICS
+};
+
+enum tap_dances {
+    TD_ESC_CAPS = 0
+};
+
+qk_tap_dance_action_t tap_dance_actions[] = {
+    [TD_ESC_CAPS]  = ACTION_TAP_DANCE_DOUBLE(KC_ESC, KC_CAPS)
+};
+
+enum custom_keycodes {
+    PLACEHOLDER = SAFE_RANGE, // can always be here
+    EPRM,
```

Comment:
This isn't actually needed anymore, as there is an official keycode for this: 
`EEP_RST`

```suggestion
```